multimodal/src/autogluon/multimodal/optimization/lit_distiller.py

```diff
@@ -154,15 +161,22 @@ def __init__(
         self.temperature = temperature
         self.hard_label_weight = hard_label_weight
         self.soft_label_weight = soft_label_weight
+        self.intermediate_loss_weight = intermediate_loss_weight
         self.hard_label_loss_func = hard_label_loss_func
         self.soft_label_loss_func = soft_label_loss_func
+        self.intermediate_loss_func = intermediate_loss_func
         if isinstance(validation_metric, BaseAggregator) and custom_metric_func is None:
             raise ValueError(
                 f"validation_metric {validation_metric} is an aggregation metric,"
                 "which must be used with a customized metric function."
             )
         self.custom_metric_func = custom_metric_func
 
+        # Adapt student's intermediate feature to teacher's
+        teacher_model_dim = self.teacher_model.text_feature_dim
+        student_model_dim = self.student_model.text_feature_dim
```

Comment:
This won’t work if you are distilling a fusion model to another fusion model.

Reply:
@zhiqiangdon Is it possible for us to get the dimensionality of the hidden features?

Reply:
I think I figured out the way to support fusion model. I'll add it soon as well as its tests.

Reply:
We add the `out_features` attribute in the fusion models.